src/template-strings/info.js

```diff
@@ -23,6 +23,8 @@ const infoMsg = {
 
   IntegrationCreate: 'Created integration on API \'{{apiPath}}\'',
 
+  IntegrationExecute: 'Executed integration on API \'{{apiPath}}\'',
```

Comment:
For this, the response we get back is
`Executed integration on API 'IntegrationsDemo/Petstore2/1.0.0/d2950470-e124-485b-827e-b649ad2e1963'`
Which I don't think is right.
Would this make more sense?
`'Executed integration \'{{integrationId}}\' on API \'{{apiPath}}\''`, you'd have to change the execute code to something like this
```this.logCommandSuccess({ integrationId, apiPath: `${owner}/${api}/${version}` })```

Reply:
It doesn’t make sense to me. If it was integration name yes that would help but integrationId is not useful information as I don't recognize it at the first look.

Reply:
apiPath is `IntegrationsDemo/Petstore2/1.0.0/d2950470-e124-485b-827e-b649ad2e1963` here. I think it's a naming issue I chanedit to `integrationPath`

Reply:
That's fair enough yeah but I don't think the `integrationId` should be on the end if you are saying `on API`. API should only be `${owner}/${api}/${version}`

Reply:
I agree that the better user experience is to display:
`'Executed integration \'{{integrationId}}\' on API \'{{apiPath}}\''`

The ID is not very friendly but it's literally the integration identifier so should be displayed.

Reply:
@barryadk But what's the value for the user here?

Reply:
To confirm that the correct integration was executed.

Reply:
So in that case we need to change `IntegrationCreate` too.

Reply:
That's fine I update it. The name could be better but ID works

Reply:
>So in that case we need to change IntegrationCreate too.

At the time the API didn't return the integration ID so it wasn't possible. I think it would be a nice improvement now that we can display the ID.